src/app.js

```diff
@@ -1,5 +1,165 @@
+  } else if (Number.isNaN(a) || Number.isNaN(b)) {
+    res.status(400).json({ error: 'Parameters "a" and "b" must be valid numbers.' });
+  } else {
+    res.status(200).json({ result: multiply(a, b) });
+  }
+});
+
+app.post('/numbers/divide', (req, res) => {
+  const a = parseInt(req.body.a, 10);
+  const b = parseInt(req.body.b, 10);
+  if (b === 0) {
+    res.status(400).json({ error: 'Unable to divide by 0.' });
+  } else if (Object.keys(req.body).length < 2) {
+    res.status(400).json({ error: 'Parameters "a" and "b" are required.' });
+  } else if (Number.isNaN(a) || Number.isNaN(b)) {
+    res.status(400).json({ error: 'Parameters "a" and "b" must be valid numbers.' });
+  } else {
+    res.status(200).json({ result: divide(a, b) });
+  }
+});
+
+app.post('/numbers/remainder', (req, res) => {
+  const a = parseInt(req.body.a, 10);
+  const b = parseInt(req.body.b, 10);
+  if (b === 0) {
+    res.status(400).json({ error: 'Unable to divide by 0.' });
+  } else if (Object.keys(req.body).length < 2) {
+    res.status(400).json({ error: 'Parameters "a" and "b" are required.' });
+  } else if (Number.isNaN(a) || Number.isNaN(b)) {
+    res.status(400).json({ error: 'Parameters must be valid numbers.' });
+  } else {
+    res.status(200).json({ result: remainder(a, b) });
+  }
+});
+
+// Booleans
+
+app.post('/booleans/negate', (req, res) => {
+  res.status(200).json({ result: negate(req.body.value) });
+});
+
+app.post('/booleans/truthiness', (req, res) => {
+  res.status(200).json({ result: truthiness(req.body.value) });
+});
+
+app.get('/booleans/is-odd/:a', (req, res) => {
+  const a = parseInt(req.params.a, 10);
+  if (Number.isNaN(a)) {
+    res.status(400).json({ error: 'Parameter must be a number.' });
+  } else {
+    res.status(200).json({ result: isOdd(a) });
+  }
+});
+
+app.get('/booleans/:string/starts-with/:char', (req, res) => {
+  if (req.params.char.length > 1) {
+    res.status(400).json({ error: 'Parameter "character" must be a single character.' });
+  } else {
+    res.status(200).json({ result: startsWith(req.params.char, req.params.string) });
+  }
+});
+
+// Arrays
+
+app.post('/arrays/element-at-index/:index', (req, res) => {
+  res.status(200).json({ result: getNthElement(req.params.index, req.body.array) });
+});
+
+app.post('/arrays/to-string', (req, res) => {
+  res.status(200).json({ result: arrayToCSVString(req.body.array) });
+});
+
+app.post('/arrays/append', (req, res) => {
+  res.status(200).json({ result: addToArray2(req.body.value, req.body.array) });
+});
+
+app.post('/arrays/starts-with-vowel', (req, res) => {
+  res.status(200).json({ result: elementsStartingWithAVowel(req.body.array) });
+});
+
+app.post('/arrays/remove-element', (req, res) => {
+  if (req.query.index) {
+    res.status(200).json({ result: removeNthElement2(req.query.index, req.body.array) });
+  } else {
+    res.status(200).json({ result: removeNthElement2(0, req.body.array) });
+  }
```

Comment:
While this works, theres a bit of repetition here that isnt needed. You could simplify this with a ternary operator like this:
```
  res
    .status(200)
    .json({ result: removeNthElement2(req.query.index ? req.query.index : 0, req.body.array) });
```

Going a bit further, you could also use a [null coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) and phrase the same things thing like this:
```
  res
    .status(200)
    .json({ result: removeNthElement2(req.query.index ?? 0, req.body.array) });
```